.github/scripts/olm_test.sh

```diff
@@ -4,5 +4,7 @@ set -e
 TAG=${TAG:-latest}
 export TEMPLATES=${PWD}/templates/build/enmasse-${TAG}
 
+export OCP4_EXTERNAL_IMAGE_REGISTRY=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' image-registry):5000
```

Comment:
Given that this is only run on minikube, does it need to be set? Or should we name it CLUSTER_EXTERNAL_IMAGE_REGISTRY?

Reply:
yes yes it need to be set, it makes the trick so the push to image registry works.
Rename it to CLUSTER_EXTERNAL_IMAGE_REGISTRY could make sense